src/app/components/experiment/experiment.component.ts

```diff
@@ -172,12 +170,25 @@ export class ExperimentComponent implements OnInit, OnDestroy {
         });
     }
 
-    async changeDefaultTemplate(templateId: string | null) {
+    showSelectDefaultTemplateDialog() {
+        const dialogRef = this.dialog.open(ChooseTemplateComponent, {
+            minWidth: "20rem", maxWidth: "40rem", width: "60%", maxHeight: "95%",
+            data: this.uiTemplate
+        });
+        dialogRef.afterClosed().subscribe(templateId => {
+            if (templateId != null) {
+                const id = templateId === 'all-plugins' ? null : templateId;
```

Comment:
There is now a constant for the special `all-plugins` value (I believe in the templates service).